.github/workflows/e2e.yml

```diff
@@ -136,7 +136,7 @@ jobs:
       - name: Download and Install Bright CLI using MSI
         shell: pwsh
         run: |
-          Invoke-WebRequest -Uri https://github.com/NeuraLegion/bright-cli/releases/download/${{ inputs.version }}/bright-cli.msi -OutFile bright-cli.msi
+          Invoke-WebRequest -Uri 'https://github.com/NeuraLegion/bright-cli/releases/download/${{ inputs.version }}/bright-cli.msi' -OutFile bright-cli.msi
```

Comment:
It looks like that [helped out](https://github.com/NeuraLegion/bright-cli/actions/runs/15272383531/job/42950803383):
<img width="1330" alt="image" src="https://github.com/user-attachments/assets/b372d052-f23d-430f-8cc4-6d1378d76eab" />